ribbon/src/main/java/feign/ribbon/LoadBalancingTarget.java

```diff
@@ -45,26 +45,38 @@
 
   private final String name;
   private final String scheme;
+  private final String path;
   private final Class<T> type;
   private final AbstractLoadBalancer lb;
+  
+  @Deprecated
   protected LoadBalancingTarget(Class<T> type, String scheme, String name) {
     this.type = checkNotNull(type, "type");
     this.scheme = checkNotNull(scheme, "scheme");
     this.name = checkNotNull(name, "name");
+    this.path = "";
+    this.lb = AbstractLoadBalancer.class.cast(getNamedLoadBalancer(name()));
+  }
+  
+  protected LoadBalancingTarget(Class<T> type, String scheme, String name, String path) {
+    this.type = checkNotNull(type, "type");
+    this.scheme = checkNotNull(scheme, "scheme");
+    this.name = checkNotNull(name, "name");
+    this.path = checkNotNull(path, "path");
     this.lb = AbstractLoadBalancer.class.cast(getNamedLoadBalancer(name()));
   }
 
   /**
-   * creates a target which dynamically derives urls from a {@link com.netflix.loadbalancer.ILoadBalancer
+   * Creates a target which dynamically derives urls from a {@link com.netflix.loadbalancer.ILoadBalancer
    * loadbalancer}.
    *
    * @param type       corresponds to {@link feign.Target#type()}
-   * @param schemeName naming convention is {@code https://name} or {@code http://name} where name
+   * @param scheme     naming convention is {@code https://name/api/v2} or {@code http://name/context} where name
```

Comment:
or just call it url (since we parse it into those things..). At any rate, keep the base example in the javadoc (ie without a path) along with the other ones.